pkgs/development/compilers/unison/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, autoPatchelfHook
+, zlib, ncurses, gpm
```

Comment:
Sorry, forgot to mention you have to add these to
`buildInputs`.

Reply:
OK, done.  Still builds on osx, except i can't have `gpm` as a `buildInput` since it's not supported on osx.